index.html

```diff
@@ -519,19 +524,19 @@ <h2>Signature Suites</h2>
         <dd>
 A URL that identifies the <a>canonicalization algorithm</a> to use on the
 <var>document</var>. For example:
-<code>https://w3id.org/security#GCA2015</code>.
+<code>https://w3id.org/security#URDNA2015</code>.
         </dd>
         <dt>digestAlgorithm</dt>
         <dd>
 A URL that identifies the <a>message digest algorithm</a> to use on the
 <var>canonicalized document</var>. For example:
-<code>https://www.ietf.org/assignments/jwa-parameters#SHA256</code>
+<code>http://w3id.org/digests#sha512</code>
```

Comment:
This should still be SHA256, right?

Reply:
Also, why did this URL change; that URL doesn't exist and it's not `https`...

Reply:
> Also, why did this URL change

I took it from here: https://github.com/w3c-dvcg/security-vocab/blob/98a86ead42f6a479c7d33938da6cfb0adc0695db/index.html#L581

(Changed it because `https://www.ietf.org/assignments/jwa-parameters#SHA256` doesn't exist either; will revert it though.)